products/postmarketos.md

```diff
@@ -0,0 +1,75 @@
+---
+title: postmarketOS
+category: os
+tags: linux-distribution
+#iconSlug: postmarketOS
+permalink: /postmarketos
+versionCommand: cat /etc/os-release
+releasePolicyLink: https://wiki.postmarketos.org/wiki/Releases
+eolColumn: Security Support
+eolWarnThreshold: 7
```

Comment:
We can increase this to whatever value you see fit. My original thought was that a relase is typically only supported for `200` -ish days or so, and the default `121` would warn at roughly half time when the new release is not much closer than the old.

Reply:
~2 months of warning period should be good, you get one month advance warning, and another month to upgrade after the next release. So `60` perhaps?

Given current policy, we won't get the advance warning (since EOL is set on next release), but 60 seems good given current release cadence.

Reply:
`60` is also good from my side.